ffmpeg/plan.sh

```diff
@@ -34,15 +46,25 @@ do_build() {
     --disable-static \
     --enable-avisynth \
     --enable-avresample \
+    --disable-stripping \
+    --enable-fontconfig \
     --enable-gmp \
+    --enable-gnutls \
     --enable-gpl \
+    --enable-libdrm \
+    --enable-libfreetype \
     --enable-libopenjpeg \
     --enable-libwebp \
+    --enable-libxml2 \
     --enable-shared \
     --enable-version3
   make -j"$(nproc)"
 }
 
 do_check() {
+  local ffmpeg_path="$HAB_CACHE_SRC_PATH/${pkg_dirname}"
+  export LD_LIBRARY_PATH="$LD_LIBRARY_PATH":"${ffmpeg_path}/libavutil":"${ffmpeg_path}/libavcodec":"${ffmpeg_path}/libswresample"
+  export LD_LIBRARY_PATH="$LD_LIBRARY_PATH":"${ffmpeg_path}/libavdevice":"${ffmpeg_path}/libavfilter":"${ffmpeg_path}/libavformat"
+  export LD_LIBRARY_PATH="$LD_LIBRARY_PATH":"${ffmpeg_path}/libavresample":"${ffmpeg_path}/libpostproc":"${ffmpeg_path}/libswscale"
   make check
 }
```

Comment:
We should call `unset LD_LIBRARY_PATH` after `make check` to indicate the scope of when we want this variable set.